test/e2e/s2i-e2e.bats

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bats
+
+source ./test/helper/helper.sh
+
+# E2E tests parameters for the test pipeline
+readonly E2E_PARAM_PATH_CONTEXT="${E2E_PARAM_PATH_CONTEXT:-.}"
+readonly E2E_PVC_NAME="${E2E_PVC_NAME:-}"
+readonly E2E_S2I_IMAGE="${E2E_S2I_IMAGE:-}"
+
+
+
+# Spinning up a PipelineRun using the S2I Go Task to build and push a container image
+@test "[e2e] pipeline-run using s2i-go task" {
+  # Asserting required configuration is informed
+  [ -n "${E2E_PARAM_PATH_CONTEXT}" ]
+  [ -n "${E2E_PVC_NAME}" ]
+  [ -n "${E2E_S2I_IMAGE}" ]
+  [ -n "${E2E_S2I_TLS_VERIFY}" ] 
```

Comment:
Is it required to be set to `false` during testing?

Reply:
Local container registry cannot be used during the e2e testing unless the tls-verification is set to false. So i think it's necessary.

Reply:
Cool, this goes along with the Makefile's "top comment".